### ISSUE TITLE
Add color variants to products

### ISSUE THREAD

**Issue by kimdv** [BalestraPatrick/HomeKitty#116]
Some products (As HomePod) can hav e multiple colors. It would be nice if we could add color variants or multiple images.

**Comment by BalestraPatrick** [comment]
Yeah it's a good idea, we could also simply have multiple images for each product. This would probably require a separate table to lookup them up.

**Comment by kimdv** [comment]
Very product should have a "display" image. 
And then some "alternatives", like colors, diffrent angles maybe ?

**Comment by BalestraPatrick** [comment]
I was thinking to simply have a slideshow of images, so people can see more images before deciding to open the product.
